README.md

```diff
@@ -65,10 +67,34 @@ on how to do so.
 
 ## Usage
 
-Using material-ui components is very straightforward.
-Once material-ui is included in your project, you can use the components this way:
+Material-UI components are easy to use. The quickest way to get up and running is by using the `MuiThemeProvider` to inject the theme into your application context. Following that, you can to use any of the components as demonstrated in our documentation.
 
-```js
+Here is a quick example to get you started:
+
+**./App.js**
+```jsx
+import React from 'react';
+import ReactDOM from 'react-dom';
+import getMuiTheme from 'material-ui/styles/getMuiTheme';
+import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
+import MyAwesomeReactComponent from './MyAwesomeReactComponent';
+
+const muiTheme = getMuiTheme();
+
+const App = () => (
+  <MuiThemeProvider muiTheme={muiTheme}>
```

Comment:
could we not just use ` muiTheme={getMuiTheme()}` directly here? App should only render once, right?